content/blog/2022/11/2022-11-17-jenkins-election-candidates.adoc

```diff
@@ -0,0 +1,131 @@
+---
+:layout: post
+:title: "Jenkins Elections Announcement"
+:tags:
+- community
+- governance
+- governance-board
+- elections
+authors:
+- kmartens27
+description: >
+    Announcement for Jenkins Board elections 2022.
+links:
+  discourse: true
+opengraph:
+  image: /images/governance/elections/2022/2022-opengraph.png
+---
+
+Hello all, voter registration and candidate nominations for the 2023 Jenkins elections are now complete.
```

Comment:
```suggestion
Hello all, voter registration and candidate nominations for the 2022 Jenkins elections are now complete.
```

Reply:
Agreed that is the correct year.  I forgot that the election year is defined by the year when it happens, not by the year when the elected individuals serve.  See https://www.jenkins.io/blog/2022/10/20/jenkins-election-announcement/